File: Ruby/CountLetterInStringList.rb
words = ['aaaa', 'dupa', 'ala ma kota']

def count_letter_in_string(words, sign)
  words.inject(0){|acc, x| x.chars.to_a.select{|letter| letter == sign }.length + acc}
end

puts(count_letter_in_string(words, 'a'))

File: Elixir/countletterinstringlist.ex
word_list = ["aaaa", "dupa", "ala ma kota"]

defmodule WordUtils do
  def word_filter(words, letter) do
    words |> Enum.map(fn x -> String.split(x, "") end) |> Enum.reduce(fn x, acc -> acc ++ x end) |> Enum.filter(fn x -> x == letter end) |> Enum.count
  end
end

IO.inspect WordUtils.word_filter(word_list, "a")
